src/contractor/contractor.cpp

```diff
@@ -1065,17 +1045,15 @@ Contractor::WriteContractedGraph(unsigned max_node_id,
 
     const std::uint64_t node_array_size = node_array.size();
     // serialize crc32, aka checksum
-    hsgr_output_stream.write((char *)&edges_crc32, sizeof(unsigned));
+    hsgr_output_file.WriteOne(edges_crc32);
     // serialize number of nodes
-    hsgr_output_stream.write((char *)&node_array_size, sizeof(std::uint64_t));
+	hsgr_output_file.WriteElementCount64(node_array_size);
     // serialize number of edges
-    hsgr_output_stream.write((char *)&contracted_edge_count, sizeof(std::uint64_t));
+	hsgr_output_file.WriteElementCount64(contracted_edge_count);
     // serialize all nodes
     if (node_array_size > 0)
     {
-        hsgr_output_stream.write((char *)&node_array[0],
-                                 sizeof(util::StaticGraph<EdgeData>::NodeArrayEntry) *
-                                     node_array_size);
+		hsgr_output_file.WriteFrom(&node_array, node_array_size);
```

Comment:
This seems to trigger the trivially copyable static assertion on Travis

https://travis-ci.org/Project-OSRM/osrm-backend/jobs/187703561#L2237-L2241